rclcpp/test/test_node.cpp

```diff
@@ -1445,9 +1474,10 @@ TEST_F(TestNode, set_parameters_atomically_undeclared_parameters_not_allowed) {
       };
     node->set_on_parameters_set_callback(on_set_parameters);
 
-    auto ret = node->set_parameters_atomically({
+    auto ret = node->set_parameters_atomically(
+    {
       {name1, 2},
-      {name2, false},  // should fail to be set, failing the whole operation
+      {name2, false},   // should fail to be set, failing the whole operation
```

Comment:
Why is this necessary?

Same below.

Reply:
I'm not sure how that snuck in there, but of course I blame uncrustify.